data/Zeppel_2008/metadata.yml

```diff
@@ -0,0 +1,163 @@
+    with the stem) were measured. Three branches from each tree were selected from
+    the lower-, mid- and upper-third of the canopy.  For sap flow measurement and
+    transpiration, a minimum of seven and a  maximum of 15 trees, that represented
+    the range of size classes present at the study site, were instrumented with commercial
+    sap flow sensors for each species at each sampling time.
+  original_file: none, data extracted from paper.
+  notes: data extracted from paper, not raw data.
+locations:
+  Liverpool Plains:
+    latitude (deg): -31.5
+    longitude (deg): 150.7
+    description: open woodland
+    soil type: well drained, acidic, lithic bleached earthy sands with pockets of
+      clay.
+    elevation (m): 390
+contexts:
+- context_property: sampling season
+  category: temporal
+  var_in: season
+  values:
+  - value: winter 2002
+    description: Measurements taken in winter 2002.
+  - value: summer 2002-2003
+    description: Measurements taken in summer 2002-2003.
+  - value: winter 2003
+    description: Measurements taken in winter 2003.
+  - value: summer 2003-2004
+    description: Measurements taken in summer 2003-2004.
+traits:
+- var_in: huber_value
+  unit_in: m2{sapwood}/m2{leaf}
+  trait_name: huber_value
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: replicates_huber
+  methods: Three branches from each tree were selected from the lower-, mid- and upper-third
+    of the canopy and the total projected leaf area of each branch was determined
+    with a leaf area meter (WinDIAS Delta-T Devices Ltd, Burwell, Cambridge, UK).
+    C. glaucophylla leaves were examined under a binocular microscope, and were found
+    to be three-sided. The projected area of photosynthetically active material was
+    measured only on C. glaucophylla. A regression of branch diameter and leaf area
+    for each species was then used to calculate the total leaf area per tree from
+    the branch diameters of all branches for each individual tree. Regressions of
+    DBH v. total branch diameter and individual branch diameter v. leaf area per stem
+    were also calculated. This allowed an estimate of leaf area to be calculated from
+    DBH by allometric relationships (Vann et al. 1998; Burrows et al. 2000; Santiago
+    et al. 2000). Sapwood cross-sectional area was measured on two 5-mm-diameter cores,
+    taken from the trunk at 1.3-m height. Sapwood area was measured on each tree that
+    was instrumented with Greenspan sap flow loggers. In addition, sapwood area was
+    measured on trees that were felled. A distinct colour change was observed between
+    bark, sapwood and heartwood in both species. Huber value (Hv) was calculated by
+    dividing sapwood area (m2) by leaf area (m2).
+- var_in: maximum_stomatal_conductance
+  unit_in: mmol{H2O}/m2/s
+  trait_name: leaf_stomatal_conductance_per_area_ambient
```

Comment:
In this case, because conductance was measured on naturally growing leaves without any supplemental light. When measurements are made with a LiCor 6400 (or similar) it is much more common for gas exchange measurements to be made under saturating light.

Reply:
Thank you!